apps/site/assets/ts/stop/components/amenities/ParkingAmenityCard.tsx

```diff
@@ -117,18 +119,71 @@ const getModalContent = (
   );
 };
 
+const getBadge = (
+  parkingLots: ParkingLot[],
+  areAllParkingLotsClosed: boolean
+): JSX.Element => {
+  if (parkingLots.length === 0) {
+    return <Badge text="Not available" bgClass="u-bg--gray-lighter" />;
+  }
+  if (areAllParkingLotsClosed) {
+    return <Badge text="Temporarily closed" />;
+  }
+  return <></>;
+};
+
+const allParkingLotsClosed = (
+  alerts: Alert[],
+  parkingLots: ParkingLot[]
+): boolean => {
+  if (parkingLots.length === 0) {
+    return false;
+  }
+  const parkingClosuresIDs = alertsForEffect(alerts, "parking_closure").flatMap(
+    a => a.informed_entity.facility
+  );
+  const parkingLotIDs = parkingLots.map(p => p.id);
+
+  // if both arrays have the same ids in them then all lots are closed
+  // meaning `openParkingLotIDs` should be empty if all lots are closed
+  const openParkingLotIDs = difference(parkingLotIDs, parkingClosuresIDs);
+
+  return openParkingLotIDs.length === 0;
+};
+
 const ParkingAmenityCard = ({
   stop,
   alertsForParking
 }: {
   stop: Stop;
   alertsForParking: Alert[];
 }): JSX.Element => {
-  const icon = <span className="m-stop-page__icon">{parkingIcon()}</span>;
+  const icon = (
+    <span className="m-stop-page__icon u-color-brand-primary">
+      {parkingIcon()}
```

Comment:
nitpick: I've been adding the classnames as arguments to these functions, e.g. `whateverIcon("some-class-name")`, which puts the class names on the `<i>` element. might be worth a try if this `<span>` wrapper and `.m-stop-page__icon` are not needed.

Reply:
For some reason moving `m-stop-page__icon` into `parkingIcon("m-stop-page__icon")` caused the icon to shrink.